src/commands/utils.rs

```diff
@@ -244,3 +244,26 @@ pub fn select_coins_for_spend(
         change_amount,
     ))
 }
+
+/// An unsigned transaction's maximum possible size in vbytes after satisfaction.
+///
+/// This assumes all inputs are internal (or have the same `max_sat_weight` value).
+///
+/// `tx` is the unsigned transaction.
+///
+/// `max_sat_weight` is the maximum weight difference of an input in the
+/// transaction before and after satisfaction. Must be in weight units.
+pub fn unsigned_tx_max_vbytes(tx: &bitcoin::Transaction, max_sat_weight: u64) -> u64 {
```

Comment:
This function could be used in `sanity_check`, but I'll wait in case any changes are required here (we could also change `sanity_check` in a separate PR).